pkg/slayers/path/io_msgterm_spec.gobra

```diff
@@ -44,3 +61,24 @@ ensures  len(res) == MacLen
 ensures  forall i int :: { res[i] } 0 <= i && i < MacLen ==> mac[i] == res[i]
 decreases
 pure func FromSliceToMacArray(mac []byte) (res [MacLen]byte)
+
+
+ghost
+requires len(mac1) == MacLen
+requires forall i int :: { &mac1[i] } 0 <= i && i < MacLen ==> acc(&mac1[i], R50)
+requires forall i int :: { &mac1[i] } 0 <= i && i < MacLen ==> mac1[i] == mac2[i]
+ensures  forall i int :: { &mac1[i] } 0 <= i && i < MacLen ==> acc(&mac1[i], R50)
+ensures  AbsMac(FromSliceToMacArray(mac1)) == AbsMac(mac2)
+decreases
+func AbsMac_Lemma(mac1 []byte, mac2 [MacLen]byte) {
```

Comment:
Please rename this to `EqualBytesImplyEqualMac`, let's not use underscores here.

Also, the postcondition here should be `FromSliceToMacArray(mac1) == mac2`, which is stronger than the current post, and I think that we can simplify the proof